CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## Unreleased
+ 
+* [Fix] Rather than incorrectly relying on `ENABLE_HTTPS`, honour
+  `S3_USE_SSL` when setting `AWS_S3_ENDPOINT_URL`.
+* [Fix] Only populate the `AWS_S3_ENDPOINT_URL` Open edX setting if
+  `S3_ENDPOINT_URL` is set, and recommend leaving `S3_ENDPOINT_URL`
```

Comment:
The user doesn't directly set `S3_ENDPOINT_URL` they set `S3_HOST` and `S3_PORT` instead and we set the`AWS_S3_ENDPOINT_URL` for them.

Reply:
I can update the text.

Reply:
Ah yes, fair point. Please do. :)

Reply:
Done